functions.cmake

```diff
@@ -158,15 +158,23 @@ function(install_clr targetName)
 endfunction()
 
 function(_add_executable)
-    add_executable(${ARGV})
+    if(NOT WIN32)
```

Comment:
Yes they look fine. I would replace else(NOT WIN32) with else() ...so that it is not confusing.